ckanext/harvest/logic/action/harvest_additions.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Purpose: Read CSV File and create harvest sources
+# Author: James Brown
+# ------------------------------------------
+
+# Display Help
+Help()
+{
+    echo "Adds harvest sources from a CSV file"
+    echo
+    echo "Syntax: harvest_additions [-f|h|k|o]"
+    echo "options:"
+    echo "[required] -f, --file             Path to CSV file"
+    echo "-h, --help                        Print this Help"
+    echo "[required] -k, --api-key          CKAN API key"
+    echo "[required] -o, --organization     CKAN organization name"
+    echo "-u, --url                         Base URL for CKAN instance"
+    echo
+    exit 0
+}
+
+#Default value for testing
+URL='localhost:8080'
+
+#Argument parsing
```

Comment:
I would use a tool like [getopts](https://www.unix.com/man-page/linux/1/getopts/) to help with the argument parsing instead of having to write your own.

Reply:
I looked at that, but thought that the less stuff I had to pull in the better: so I went simple.